packages/slice-machine/src/pages/changes.tsx

```diff
@@ -84,7 +96,11 @@ const Changes: React.FunctionComponent = () => {
         // Update last sync value in local storage
         pushChangesSuccess();
 
-        toast.success("All slices and types have been pushed");
+        // TODO: hide this toast for users in experiment eligible version (DT-2287)
+        setIsToastOpen(true);
+
+        // TODO: display this toast for users in experiment control version (DT-2287)
+        // toast.success("All slices and types have been pushed");
```

Comment:
I'm confused about these two todos, is actually the same todo, no? why not add the flag right away? (or add the flag first, then add the toast second)

that way we keep the project in a shippable state

Reply:
there is still discussion about the experiment management open in the second PR, that's why I didn't include it here yet
since we want the new Toast to be a default one I thought is ok like that